src/Webroot.php

```diff
@@ -43,12 +43,12 @@ public function sniff() {
 	 * Execute a command on the relevant server, using the given sudo option
 	 * @param  string $command Shell command, either a fully escaped string or an array
 	 */
-	public function execSudo($command) {
+	public function execSudo($command, $options = array()) {
```

Comment:
Please update the docblock to reflect the additional parameter.

Reply:
@dhensby are we consciously not passing `$options` through via sudo for any reason, security or otherwise?

Reply:
🤷‍♂️ I'm not sure :/